aiidalab_eln/base_connector.py

```diff
@@ -14,8 +14,6 @@ def __init__(self, **kwargs):
 
         Args:
             eln_instance (str): URL which points to the eln instance.
-            user (str): ELN user.
-            token (str): ELN access token (or user password).
             eln_type (str): ELN type, e.g. "cheminfo".
```

Comment:
```suggestion
            eln_type (str): ELN type, e.g. "cheminfo" or "openbis".
```

maybe we could even make this an `enum`